values.yaml

```diff
@@ -180,7 +180,7 @@ persistence:
     # backends which not supported it (such as using minio for `s3` storage type), please disable
     # it. To disable redirects, simply set `disableredirect` to `true` instead.
     # Refer to
-    # https://github.com/distribution/distribution/blob/main/docs/configuration.md#redirect
+    # https://github.com/distribution/distribution/blob/main/docs/content/about/configuration.md#redirect
```

Comment:
since Harbor is currently using distribution 2.8, maybe changing the URL to:
* https://github.com/distribution/distribution/blob/release/2.8/docs/configuration.md#redirect